linalg/sundials.cpp

```diff
@@ -397,14 +528,15 @@ N_Vector SundialsNVector::MakeNVector(MPI_Comm comm, bool use_device)
       if (use_device)
       {
          x = N_VMake_MPIPlusX(comm, N_VNewWithMemHelp_Cuda(0, UseManagedMemory(),
-                                                           sunmemHelper));
+                                                           sunmemHelper,
```

Comment:
Should `sunmemHelper` be `Sundials::GetMemHelper()`?

Reply:
Also, need a `SUNContext` argument at the end.

Reply:
Should now be addressed!